zeebe/backup/src/main/java/io/camunda/zeebe/backup/api/BackupManager.java

```diff
@@ -36,13 +36,17 @@ public interface BackupManager {
    */
   ActorFuture<BackupStatus> getBackupStatus(long checkpointId);
 
+  default ActorFuture<Collection<BackupStatus>> listBackups() {
+    return listBackups(null);
+  }
+
   /**
    * Get all available backups where status is one of {@link BackupStatusCode#COMPLETED}, {@link
    * BackupStatusCode#FAILED}, {@link BackupStatusCode#IN_PROGRESS}
    *
    * @return a collection of backup status
```

Comment:
🔧 Please update the java doc

Reply:
Done in 9f7a80aec1e156f60ab4b7961b9e433d06248df0